dll/cpl/appwiz/addons.c

```diff
@@ -321,8 +321,23 @@ static HRESULT WINAPI InstallCallback_OnStopBinding(IBindStatusCallback *iface,
 static HRESULT WINAPI InstallCallback_GetBindInfo(IBindStatusCallback *iface,
         DWORD* grfBINDF, BINDINFO* pbindinfo)
 {
-    /* FIXME */
```

Comment:
Is zeroing parameters all that is expected? Or is some TODO to get/set some actual data still wanted?

Reply:
Expected.